canonical-web-and-design/react-components
# Prepare release 0.16.0

## Done

- Updated changelog
- Updated generated docs
- Updated version to 0.16.0

## QA

- Review changelog
- Pull the branch, verify that `docs/index.html` generated docs work when opened in browser
  - check if newly added CodeSnippet docs are in generated docs
